xml/docu_styleguide.smartdocs.xml

```diff
@@ -300,10 +300,8 @@ ultrashort description for social media, max. 55 chars&lt;/meta>
 &lt;merge>
   &lt;title>Your title, limit to 55 characters, if possible&lt;/title>
   &lt;subtitle>Subtitle if necessary&lt;/subtitle>
-  &lt;revhistory>
-    &lt;title>Changelog&lt;/title>
+  &lt;revhistory xml:id="rh-MY_ROOTID">
```

Comment:
```suggestion
  &lt;revhistory xml:id="rh-USE-ROOT-ID">
```

Reply:
This was suggested by Jana in her draft template https://github.com/SUSE/doc-modular/pull/228/files

Reply:
Yes, but I'd suggest to use _underscores_ (_) on *purpose*. People copy-and-paste and forget to adapt it. If they do, validation will fail and reminds them to change that.

We could use `rh-USE_ROOT_ID` to make validation visible.